tools/topology/sof-tgl-max98357a-rt5682.m4

```diff
@@ -69,7 +69,10 @@ MUXDEMUX_CONFIG(demux_priv_1, 2, LIST(`	', `matrix1,', `matrix2'))
 # PCM99 <---- volume <---- DMIC01 (dmic 48k capture)
 # PCM100 <---- kpb <---- DMIC16K (dmic 16k capture)
 
-# Define pipeline id for sof-tgl-max98357a-rt5682.m4
+# Define pipeline id for sof-tgl-CODEC-rt5682.m4
+ifelse(CODEC, `MAX98357A', `define(SPK_PCM_NAME, `max357a-spk')')
+ifelse(CODEC, `RT1011', `define(SPK_PCM_NAME, `Speakers')')
```

Comment:
why not make them 'Speakers' for both? that should have no impact on userspace/UCM?

Reply:
@plbossart , You are right! No impact. So fix the string 'Speakers' for both.